plugins/woo-ai/src/image-background-removal/background-removal-link.tsx

```diff
@@ -66,12 +75,17 @@ export const BackgroundRemovalLink = () => {
 					.split( '/' )
 					.pop() }`,
 			} );
+
+			recordTracks( 'complete' );
 		} catch ( err ) {
 			//eslint-disable-next-line no-console
 			console.error( err );
-			setDisplayError(
-				getErrorMessage( ( err as { code?: string } )?.code ?? '' )
-			);
+			const errorCode = ( err as { code?: string } )?.code ?? '';
+			setDisplayError( getErrorMessage( errorCode ) );
+
+			recordTracks( 'error', {
+				error: errorCode,
```

Comment:
We might include the error message here as well since the `code` may not always exist.